nilearn/__init__.py

```diff
@@ -33,11 +33,21 @@
 """
 
 import gzip
+from distutils.version import LooseVersion
 
 from .version import _check_module_dependencies, __version__
 
 _check_module_dependencies()
 
+# Temporary work around to address formatting issues in doc tests
+# with NumPy 1.14. NumPy had made more consistent str/repr formatting
+# of numpy arrays. Hence we print the options to old versions.
+import numpy as np
+if LooseVersion(np.__version__) >= LooseVersion("1.14"):
+    from ._utils.testing import is_nose_running
+    if is_nose_running:
```

Comment:
This is a function, you need to call it: `if is_nose_running()`: At the moment you will always set the numpy str/repr behaviour to 1.13.